### PR TITLE
More modifiers

### PR DESCRIPTION
**Pull request by SaulDoesCode**

### CODE REVIEW

File: ui/modifiers.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   [
     "Drawing Style",
     [
-      "Cel Shading",
+      "Cell Shading",
```
Comment (cmdr2) [inline ui/modifiers.json:5]: Hi, it is actually Cel Shading :) https://en.wikipedia.org/wiki/Cel_shading

**Comment by cmdr2** [comment]
Hi @SaulDoesCode Thanks for the PR! Nice work.

Question - if you haven't already done this, can you please test that these modifiers actually change the image to the desired style?

It would be good to avoid adding modifier tags that don't have an effect. Thanks!

File: ui/modifiers.json [hunk 2]
```diff
@@ -127,24 +153,38 @@
...
       "by Caravaggio Michelangelo Merisi",
+      "by Claude Monet",
       "by David Mann",
+      "by Diego Velázquez",
+      "by Édouard Manet",
       "by Frida Kahlo",
+      "by Georgia O'Keeffe",
       "by H.R. Giger",
       "by Hayao Miyazaki",
+      "by Henri Matisse",
```
Comment (SaulDoesCode) [inline ui/modifiers.json:164]: Mattise style works I tried it

**Comment by SaulDoesCode** [comment]
I did some renders on my machine with most of the new modifiers, they seem to yield agreeable results that match the meaning, I posted a few renders on the art section of the discord. I think most of the modifiers will have some effect, I have tried some fairly obscure artists and got good results, I think it should be fine. It's up to you.

**Comment by cmdr2** [comment]
Thanks @SaulDoesCode , I've merged this in. Thanks for the PR! :)

**Pull request merged**